app/views/thinkers/show.html.erb

```diff
@@ -9,8 +9,8 @@
 </p>
 
 <p>
-  <strong>Email:</strong>
-  <%= @thinker.email %>
+ <strong>Life Dates:</strong> 
+ <%= @thinker.life_date_birth %> - <%= @thinker.life_date_death %>
```

Comment:
This isn't essential, but I'll suggest it because it's a useful concept to be aware of, and this is a straightforward example.

Sometimes we come across cases where the rendering of fields is a little more complex then just `<%= model.field %>`. In this case, the life dates are both components of the same unit of structured data, and wherever they're displayed we want to show them in the same format (`"#{thinker.life_date_birth} - #{thinker.life_date_death}"`).

We can avoid repetition (here and in the index view), and also hide this rendering complexity from the views, by creating a helper method. In this case, we could call it something like `life_dates_for` and add it to `ThinkersHelper` so that it will be available in all the relevant thinkers views. We can then render the view like this: `<%= life_dates_for(@thinker) %>`.

The complexity here isn't especially significant, but suppose you wanted to deal with a few other cases. Hypothetically, you might decide that:

* if both dates are nil, render nothing
* if only the birth date is set, render `"b. #{thinker.life_birth_date}"`
* if both dates are set, render `"#{thinker.life_date_birth} - #{thinker.life_date_death}"`

Once we reach that kind of complexity, it's definitely appropriate to create a helper method (or to use the decorator pattern instead, e.g. with https://github.com/drapergem/draper).